### ISSUE TITLE
Can't reproduce the mAP in paper

### ISSUE THREAD

**Issue by prateethvnayak** [thtrieu/darkflow#1034]
@thtrieu  .. Is it True that using Tensorflow implementation, the best mAP you can reach is ~44% ? using the pre-trained darknet weights and cfg file. and implementing the network in TF, just not able to reproduce the paper accuracy.
All the things checked -
- Same weights for corresponding layers in Darknet and TF
- same Overview implemented layers 
- same post-processing of output with nms_thresh = 0.5 and probability_thresh = 0.25
- same test-set of VOC2007 (4952 images) , and excluding `difficult` objects

Is anyone able to reproduce this accuracy in Tensorflow ?
